### PR TITLE
Fix `ci.yml` for versioned documentation

### PR DESCRIPTION
**Pull request by hyrodium**
This PR fixes https://github.com/JuliaMatrices/BandedMatrices.jl/issues/249.

### CODE REVIEW

**Comment by hyrodium** [comment]
After merging this PR, I think the following command will trigger the versioned document deployment.
```
git tag v0.17.6 v0.17.6+doc1
git push origin v0.17.6
```

**Review by jishnub — APPROVED** [review]
LGTM, can't hurt to have extra tests

**Comment by hyrodium** [comment]
I agree with that. Let's split the yaml file.

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: CI
```
Comment (jishnub) [inline .github/workflows/docs.yml]: The name should be changed to Documentation

**Pull request merged**